### PR TITLE
count symbols in string

### PR DESCRIPTION
**Pull request by Nikby53**
add count implementation and test

### CODE REVIEW

File: strings.go [hunk 1]
```diff
@@ -8,3 +8,12 @@ func Reverse(s string) string {
...
 }
+
+// Count function returns a number of symbols in the string
+func Count(s string) int {
+	count := 0
+	for range s {
+		count++
+	}
+	return count
+}
```
Comment (papahmsolo) [inline strings.go:19]: This works fine. However, string is a slice, so we can get the number of symbols by the length of it. Beware of encoding problems.

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Cool! Let's also add a go.mod file, it should be committed to the repo.

File: strings_test.go [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+}
 
+func TestCount(t *testing.T) {
+	word := "Hello"
+	expected := 5
+	actual := Count(word)
+	if actual != expected {
+		t.Errorf("Expected %v instead of %v", expected, actual)
+	}
 }
```
Comment (papahmsolo) [inline strings_test.go:32]: Let's add more test cases

**Review by papahmsolo — CHANGES_REQUESTED** [review]
Can we change the target branch for merging to `main`?

File: strings.go (hunk 1 above)
Comment (Nikby53) [reply strings.go]: Ok, i figured it out

File: strings_test.go (hunk 2 above)
Comment (Nikby53) [reply strings_test.go]: added

**Review by alextur — APPROVED** [review]

**Pull request merged**